### ISSUE TITLE
Up to date fork UE5, UE4.27+

### ISSUE THREAD

**Issue by FAUSheppy** [VSZue/DonAINavigation#20]
In the last year I have collected fixes from various forks and added some of my own. I also updated the plugin for UE5 and UE4.26, I think this repository might have been abandoned, but @VSZue: If you ever see my PR or this Issue consider merging it.

In case this repository is really abandoned, Issues & PRs in my fork are welcome, I sometimes check other forks for new developments too:

https://github.com/VSZue/DonAINavigation/compare/master...FAUSheppy:master

PR: https://github.com/VSZue/DonAINavigation/pull/19

**Comment by gamingman** [comment]
Is there a way to get this in 4.26.2?

**Comment by FAUSheppy** [comment]
> 

What doesn't work?

**Comment by gamingman** [comment]
> > 
> 
> What doesn't work?

It was saying it could not be built try again from source. But I figured it out I am about to report the bug fix on the other comment as well. But in case anyone finds this one first and is using PerForce as source control and they hit clean. They need to mark everything as writable in the root folders property settings and check mark include subfolders.

**Comment by sbseltzer** [comment]
Any updates recently? How does this compare to https://github.com/Takexito/DonAINavigationUE5 ? It seems to have some of your commits in it.

**Comment by gamingman** [comment]
It's been a long time but you can still get this to work in UE5. by changing the project version in the settings. Word of Caution, BackUp Your Orignal Prior Or make a Clone Project within the Epic Launcher prior to changing Engine Versions.

>

**Comment by FAUSheppy** [comment]
Project has spread out into many forks. You have to change more than just the Engine Version though as you can see from the commits. I haven't checked this other fork you linked, but on a preliminary glance it seems to be mostly the same. I will check if I should port back some changes from there as soon as I have time.

**Comment by sbseltzer** [comment]
Yeah it seems to be almost identical. I had to make a change to one line to get it working with 4.27. I maintain 2 forks with a number of other fixes/improvements. The one with the most changes was made for a client project though so I can't share those without permission. At some point I'll see if I can add those improvements to my public fork and report back.

As a side note, I'm pretty sure that if you remove the engine version line from the uplugin file it will be treated as engine version agnostic. You may need to run a command to package it but all you have to do is drag and drop most of the time (at least for C++ projects). See this plugin for example: https://github.com/AutoSizeComments/AutoSizeComments/
